tests/providers/sftp/hooks/test_sftp_outdated.py

```diff
@@ -1,340 +0,0 @@
-#
```

Comment:
This file is redundant with tests/providers/sftp/hooks/test_sftp.py !! only difference was what conn_id was passed, which is taken care off by new added test -> `test_deprecation_ftp_conn_id`

Reply:
This was my mistake! Thanks a lot for rescuing